run_node.sh

```diff
@@ -77,6 +82,28 @@ done
 
 ALEPH_VERSION=$(cat env/version)
 
+if [ ! -d "${HOST_BASE_PATH}/${DB_SNAPSHOT_PATH}" ] && [ -d "${DB_SNAPSHOT_PATH}/keystore" ]
+then
+    echo "The default location of the data directory has changed."
+    echo "Your files will be copied automatically to ${HOST_BASE_PATH}/${DB_SNAPSHOT_PATH}."
+    echo "If you wish to customize the directory, select \'n\' and re-run the script"
```

Comment:
Here it could be just `'n'`, without `\` same for other two places with `--data_dir`